src/views/ConsentContinuationOut.vue

```diff
@@ -267,8 +328,7 @@ export default class ConsentContinuationOut extends Vue {
 
   /** Maximum length of detail comment. */
   get maxDetailCommentLength (): number {
-    // = (max size in db) - (default comment length) - (Carriage Return)
-    return 2000 - this.defaultComment.length - 1
+    return 1000
```

Comment:
Are you sure the default comment isn't part of this length?

Reply:
I'm not really sure to be honest. From the UXPin, I saw it was 0 / 1000. I interpreted that it's not. Maybe I'm wrong here?

Reply:
See next comment 😁

Reply:
Thanks for fixing this.